### PR TITLE
I1648 canvas items on report

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/client/react/canvas/components/CanvasReportItemList.js [hunk 1]
```diff
@@ -49,13 +50,14 @@ const CanvasReportItemList = ({
   items,
   isSimple,
   renderItem,
+  sort = identity,
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReportItemList.js]: Why not just default to sortByTitle?

Comment (Serhiy91) [reply app/imports/client/react/canvas/components/CanvasReportItemList.js]: Because "CanvasReportItemList" is used in other components.

**Comment by Dartv** [comment]
Shouldn't you also delete 2 other "Canvas items" sections?

**Comment by Serhiy91** [comment]
No. https://github.com/Pliohub/Plio/issues/1648 .
Michael has agreed with it.

**Comment by Dartv** [comment]
"the bottom one of the three different treatments shown in the prototype UI - see below"
So, this one 
And delete the other two

**Pull request merged**